frontend/ui/widget/inputdialog.lua

```diff
@@ -391,6 +389,7 @@ function InputDialog:init()
         scroll_callback = self._buttons_scroll_callback, -- nil if no Nav or Scroll buttons
         scroll = true,
         scroll_by_pan = self.scroll_by_pan,
+        is_fullscreen = self.fullscreen,
         has_nav_bar = self.add_nav_bar,
```

Comment:
These 2 properties feels (now?) odd. Their naming as properties of InputText feels semantically incorrect.
I was going to suggest to pass directly `allow_hiding_keyboard = not (self.fullscreen or self.add_nav_bar)`, but then, below, it would seem to only be used when invoking `function self:InputDialog:onTap() self._input_widget:onHideKeyboard()`.
So, you might as well forget about all these properties, and just use:
```lua
function InputDialog:onTap()
    if self.fullscreen or self.add_nav_bar then
        return
    end
    self._input_widget:onHideKeyboard()
end
```

Reply:
Oh, yes, `onHideKeyboard` previously was used in virtualkeyboard.lua for down-arrow, it is removed now.